docs/user-guide/common-operations.md

```diff
@@ -17,4 +48,33 @@ The backend data can be wiped by using:
 kubehound backend wipe
 ```
 
-These commands will reboot backend services and wipe all data.
+!!! Warning
+    This command will **wipe ALL DATA and will not be recoverable**.
```

Comment:
Can you define "ALL" (docker volume? mounted directories ? containers etc...?)?

Reply:
all docker info (docker volume and containers). updating the doc